master/static/srv/proto_get_trials_plus.sql

```diff
@@ -11,7 +11,8 @@ WITH searcher_info AS (
           ELSE -1
         END
     ) AS sign,
-    t.id AS trial_id
+    t.id AS trial_id,
+  (config->>'max_restarts')::int AS max_restarts
```

Comment:
**nit:** is there a linter for `.sql` file? should `(config...` line up with `t.id...`? :D